### ISSUE TITLE
select within toolbar disappears with sample themes

### ISSUE THREAD

**Issue by nkwood** [angular/components#4886]
#### Bug, feature request, or proposal:

Bug. (Assuming this is something that is intended to be supported)

#### What is the expected behavior?

Select placed within toolbar with sample themes should be colored as to be visible.

#### What is the current behavior?

Select placed within toolbar with sample themes is not visible.

#### What are the steps to reproduce?

Plunker demonstrating issue: http://plnkr.co/edit/7Nd4YFZ05mdYt96yp5ir?p=preview

#### What is the use-case or motivation for changing an existing behavior?

Support placement of selects within toolbars for the default themes, e.g.:
<img width="383" alt="screen shot 2017-05-30 at 3 15 41 pm" src="https://cloud.githubusercontent.com/assets/6249779/26603140/6b016780-454b-11e7-8bfd-b0f840da2f5e.png">

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

At least through @angular/material@2.0.0-beta.6 and Chrome 60.0.3107.4.

#### Is there anything else we should know?

This might be too special a use case to support, requiring the user to manage the styles for the select in this instance, but given previous work to get the color of select right with the themes (e.g. https://github.com/angular/material2/pull/4883, https://github.com/angular/material2/pull/4568) this would be nice to have.

**Comment by tomwanzek** [comment]
Is there an update on this issue? I have the same requirement, the use of `mat-select` inside a `mat-toolbar`.

IMHO the ability to use `mat-select` within a toolbar is fundamental.

The issue remains the same, if I wrap the `mat-select` into `mat-form-field`. The colors of the select element are not readily controllable to work within the toolbar. See also issue #9513, which was just closed due to an issue template formality.

**Comment by akanieski** [comment]
Any updates? looking for the same.

**Comment by SieBrum** [comment]
Any updates yet?

**Comment by JaimeStill** [comment]
Agree that any `<mat-form-field>` element should natively fit inside of a `<mat-toolbar>`.  The work around I have for now is as follows:

```scss
mat-toolbar.mat-toolbar mat-form-field.mat-form-field {
  font-size: 12px;

  label.mat-form-field-label {
    font-weight: bold;
  }
}
```  

Which will cause a `<mat-select>` to look as follows:  

![image](https://user-images.githubusercontent.com/14102723/64555366-32174e80-d30b-11e9-94e5-db08cf41a6c5.png)  

### [Toolbar Select StackBlitz](https://toolbar-select.stackblitz.io/home)  
* [material-style.scss](https://stackblitz.com/edit/toolbar-select?file=src%2Ftheme%2Fmaterial-styles.scss) - See line **107**.
* [app.component.html](https://stackblitz.com/edit/toolbar-select?file=src%2Fapp%2Fapp.component.html) - See line **6**.

**Comment by carlos-dubon** [comment]
Will this be implemented in a near future? I'm looking forward to see this as a native feature.

**Comment by thw0rted** [comment]
I think this might be the same issue as #10622 which was supposed to be fixed in #10627 .  I can't find an example of a Select, specifically, in [the "app bar" spec](https://material.io/components/app-bars-top#specs).  (There isn't a "toolbar" spec, and I think "app bar" is closest.)

**Comment by jelbourn** [comment]
Closing since this seems to work fine for the `<mat-select>`.

**Comment by thw0rted** [comment]
The screenshot in the OP is how this works today, *without* a `mat-form-field`.  Wrapping the select in a `mat-form-field` does not work.  [Stackblitz example](https://stackblitz.com/edit/components-issue-bkkdag?file=src/app/example-component.html)

I can't find any examples where you'd want to use any control in a `mat-form-field` inside an App Bar, in the spec I linked above.  As I pointed out, there's no example of any select in an App Bar either (with or without form-field wrapper), but the OP has a good screenshot of a valid use case for a bare select in the wild.  I can't think of a case where you'd want a full-fledged form field -- which can be huge, vertically, with a floating placeholder above and hint text below -- in such a tight spot.  Remember, density is coming when MDC launches.
